eth/tracers/native/call.go

```diff
@@ -153,8 +153,6 @@ func (t *callTracer) CaptureStart(env vm.VMInterface, from libcommon.Address, to
 // CaptureEnd is called after the call finishes to finalize the tracing.
 func (t *callTracer) CaptureEnd(output []byte, gasUsed uint64, err error) {
 	t.callstack[0].processOutput(output, err)
-	t.logIndex = 0
```

Comment:
i don't understand how crush at file `bor.go:152` can be related to this lines in this file